src/explore-education-statistics-frontend/.env.production

```diff
@@ -0,0 +1 @@
+PUBLIC_URL=https://explore-education-statistics.service.gov.uk/
```

Comment:
Think we'll need to update the environment variables in the Playwright tests too?

The `PUBLIC_URL` variable will be be set to the _actual_ environment URL, so imagining we'll need a `PROD_PUBLIC_URL` variable or similar.

Reply:
Hrm yes good point, but actually I wonder if we should make these tests prod only? 

So with these changes, if frontend is ran locally in dev mode we should still get a localhost robots file so the tests should pass as they are. If frontend is ran in prod mode then they'll fail. 

If we make the tests always use the live site prod url, then at each stage of the pipeline we'll effectively be running these tests against prod. If this got merged and deployed then those tests failed for any reason, they'd fail on every environment which obviously is a bit misleading and hard to get a fix through the pipeline steps.

As a side note, in my local branch for improving how we test new redirects I've introduced tags to the playwright tests so I can filter which ones are ran. The intention being to replicate the way robot tests can be ran on selected environments. We could do something similar here? 
Or, just skip these tests for now for the sake of getting a fix out, and do it in another PR?

Reply:
Making them prod only might be a good idea. I'm just a little concerned about spending too much time improving the Playwright tests given we're not committing to them for the time being.

Imagine we'll need to have some more conversations about the future of them soon, but my preference would be to have any important suites moved back into the Robot tests to keep everything cohesive.

Also, just realised we're not actually running the Playwright or Robot tests for non-dev environments in the pipeline anymore. This makes me think that adding tagging functionality probably is probably overkill.

Regardless, we might not be on the same page about the `PROD_PUBLIC_URL` variable approach. I was thinking of having this alongside `PUBLIC_URL` (it wouldn't be a replacement) and would be used for the `robots.txt` assertions like:

```ts
test('The robots.txt file points to the sitemap', async ({ page }) => {
  await page.goto(`${PUBLIC_URL}/robots.txt`);
  await expect(page).toHaveURL(`${PUBLIC_URL}/robots.txt`);

  await expect(
    page.getByText(`Sitemap: ${PROD_PUBLIC_URL}/sitemap.xml`),
  ).toHaveCount(1);

  await expect(
    page.getByText(`Sitemap: ${PROD_PUBLIC_URL}/server-sitemap.xml`),
  ).toHaveCount(1);
});
```

This way, the tests are running against their respective environment (and not only prod).

Reply:
Hrm yeah fair enough, very happy to do that 
(& agree on not spending more time on playwright - I'm only modifying existing tests / reducing the faff involved in running them, certainly not intending to add more. Certainly not doing anything that would take me longer than 15 mins or so)